pkg/app/demo.go

```diff
@@ -0,0 +1,301 @@
+
+	tea "github.com/charmbracelet/bubbletea"
+	"github.com/charmbracelet/lipgloss"
+	"github.com/cockroachdb/errors"
+	cli "github.com/urfave/cli/v2"
+)
+
+var CommandDemo = &cli.Command{
+	Name:     "demo",
+	Category: CategoryManagement,
+	Usage:    "Interactively initializes the current directory with the build.envd file",
+	Action:   demoCommand,
+}
+
+func demoCommand(clicontext *cli.Context) error {
+
+	startQuestion(LanguageChoice())
+	languageChoice := selectionMap[LabelLanguage][0]
+	if languageChoice == "python" {
+		startQuestion(PythonPackageChoice())
+	} else if languageChoice == "r" {
+		startQuestion(RPackageChoice())
+	}
+	startQuestion(CudaChoice())
+	if selectionMap[LabelCudaChoice][0] == "Yes" {
+		startQuestion(CudaVersionChoice())
+	}
+
+	err := generateFile(clicontext)
+	if err != nil {
+		return errors.Wrap(err, "error generating build.envd")
+	}
+
+	fmt.Println("Successfully generated build.envd file!")
+	return nil
+}
+
+func startQuestion(input input) {
+	p := tea.NewProgram(InitModel(input))
+	_, err := p.Run()
+	if err != nil {
+		fmt.Printf("There was an error generating build.envd: %v", err)
+		os.Exit(1)
+	}
+}
+
+var selectionMap = make(map[string][]string)
+var itemStyle = lipgloss.NewStyle().Foreground(lipgloss.Color("#999999"))
+var selectedItemStyle = lipgloss.NewStyle().Foreground(lipgloss.Color("#4dff4d"))
+
+const (
+	SINGLE_SELECT   string = "single select"
+	MULTIPLE_SELECT string = "multiple select"
+	INPUT           string = "input"
+)
+
+const (
+	LabelLanguage      string = "Language"
+	LabelCuda          string = "Cuda"
+	LabelCudaChoice    string = "Cuda Choice"
+	LabelPythonPackage string = "Python Package"
+	LabelRPackage      string = "R Package"
+)
+
+type model struct {
+	step     int
+	cursor   int
+	selected map[int]struct{}
+	input    input
+}
+
+type input struct {
+	prompt    string
+	inputType string
+	label     string
+	options   []string
+}
+
+func LanguageChoice() input {
+	return input{
+		prompt:    "Choose a programming language",
+		inputType: SINGLE_SELECT,
+		label:     LabelLanguage,
+		options: []string{
+			"python",
+			"r",
+			"julia",
+		},
+	}
+}
+
+func PythonPackageChoice() input {
+	return input{
+		prompt:    "Choose your python packages",
+		inputType: MULTIPLE_SELECT,
+		label:     LabelPythonPackage,
+		options: []string{
+			"numpy",
+			"tensorflow",
+		},
+	}
+}
+
+func RPackageChoice() input {
+	return input{
+		prompt:    "Choose your r packages",
+		inputType: MULTIPLE_SELECT,
+		label:     LabelRPackage,
+		options: []string{
+			"remotes",
+			"rlang",
+		},
+	}
+}
+
+func CudaChoice() input {
+	return input{
+		prompt:    "Include Cuda?",
+		inputType: SINGLE_SELECT,
+		label:     LabelCudaChoice,
+		options: []string{
+			"Yes",
+			"No",
+		},
+	}
+}
+
+func CudaVersionChoice() input {
+	return input{
+		prompt:    "Choose a cuda version",
+		inputType: SINGLE_SELECT,
+		label:     LabelCuda,
+		options: []string{
+			"11.0",
+			"10.2",
+			"10.1",
```

Comment:
Please update the version according to the https://github.com/tensorchord/envd/blob/main/base-images/remote-cache/build.envd

Reply:
Ok, will do!